### DISCUSSION TITLE
Integration and comparison of scATAcseq clusters between conditions

### DISCUSSION THREAD

**Discussion by ToPrinz** [stuart-lab/signac#686] [Q&A] [answered]
Dear Signac team,
I have an issue regarding the comparison of KO vs WT datasets. 
What I have: 1 scATACseq wildtype, 1scATACseq knock-out, 2 scRNAseq for WT and KO. 
I couldn´t find any pipeline for integration of these datsets following by a comparison KO vs WT in one specific cluster. 
For example, I want to know the different accessability of gene X in cluster Y between KO and WT. Is this possible somehow?

Thanks for helping

Best,
Tony

**Comment by timoast** [comment, marked as answer]
One example of how to do this is shown in the Seurat introductory integration vignette: https://satijalab.org/seurat/articles/integration_introduction.html

Basically, you can combine the cluster and treatment information into a new metadata variable (eg, `1_WT, 1_KO, 2_WT, 2_KO` etc) and then run `FindMarkers()` within a cluster between treatment groups (eg, `ident.1="1_WT", ident.2="1_KO"`)
